integration_tests/__tests__/__snapshots__/failures.test.js.snap

```diff
@@ -314,6 +316,8 @@ exports[`works with node assert 1`] = `
     Message:
       Missing expected exception.
       
+      at assert:1:1
```

Comment:
we don't want this in the stack at all, do we?

Reply:
Since these modules are now re-injected, they will appear in the stack. I don't think it's critical, though.

Reply:
it's not useful though, so why not remove it? ref #4695

Reply:
How do you decide what comes from an internal and what not? Notice that now internals are not internals anymore, because we re-evaluate them 😉

Reply:
IMO internal === not jest or node core

Reply:
Yeah, but I'm unsure on how can we remove an arbitrary number of stack frames from the stack. How are we doing that now? I can maybe adapt my filenames to the existing way.

Reply:
it's cleaned in this function: https://github.com/facebook/jest/blob/9043a44fca5b5e4880aed3579ab0c56775ec686b/packages/jest-message-util/src/index.js#L121-L163

Reply:
Yeah these guys have to go. We need a way to annotate node libs so they don't come up in stack traces.

Reply:
@SimenB this looks promising! I can inject native modules with `native` on the file name, and I think I can get rid of them.